pykechain/models/property_reference.py

```diff
@@ -49,7 +49,9 @@ def _retrieve_objects(self, **kwargs) -> List[Scope]:
 
         scopes = []
         if scope_ids:
-            scopes = list(self._client.scopes(id__in=",".join(scope_ids), status=None))
+            scopes = list()
+            for chunk in self.chunks(scope_ids):
+                scopes += list(self._client.scopes(id__in=",".join(chunk), status=None))
```

Comment:
scopes.extend() is faster than concatenation.

```suggestion
                scopes.extend(list(self._client.scopes(id__in=",".join(chunk), status=None)))
```